tests/utils/inputparams/testparams.go

```diff
@@ -42,8 +43,13 @@ func GetVolumeName() string {
 }
 
 // GetVolumeNameWithTimeStamp prepares unique volume name by appending current time-stamp value
-func GetVolumeNameWithTimeStamp(volName string) string {
```

Comment:
This is kind of useful util while debugging failure to search for testcase customized string .. 

As discussed offline please keep this util.